lib/src/main/kotlin/com/swmansion/starknet/data/SelectorFromName.kt

```diff
@@ -1,5 +1,3 @@
-@file:JvmName("Selector")
-
```

Comment:
Removing it makes it not Java friendly. Now it is used like this:

`SelectorFromNameKt.selectorFromName(x)`

Reply:
Oh, if it includes the `Kt` at the end of each package, should we be adding JvmName annotation to all files?

Reply:
Not to all files. I'm not 100% sure how all of it works, but if the file contains only classes, interfaces and objects, this annotation is not needed. However, when there are some static functions which we would like to expose, we should add the annotation to avoid Kt suffix.